cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -529,6 +548,20 @@ public KafkaBridgeHttpConfig getHttp() {
         return this.http;
     }
 
+    /**
+     * @return The Consumer configuration of the Bridge
+     */
+    public KafkaBridgeConsumerSpec getKafkaBridgeConsumer() {
+        return this.kafkaBridgeConsumer;
+    }
+
+    /**
+     * @return The Producer configuration of the Bridge
+     */
+    public KafkaBridgeProducerSpec getKafkaBridgeProducer() {
+        return this.kafkaBridgeProducer;
+    }
```

Comment:
I don't think these should be needed.

Reply:
Needed for testing, an alternative will be to set `this.kafkaBridgeConsumer` and `this.kafkaBridgeProducer` as protected.

Reply:
Yeah, but these obejects are just _passed through_ as they are in the custom resource. So that test IMHO does not add any real value. The important part to test is that the environment variables are set accordingly and not that the object passed in the `fromCrd` method still looks the same. so I think the checks of these objects ae not really needed in the tests.

Reply:
Yeah I would agree with Jakub on this.